.cargo/config.toml

```diff
@@ -33,15 +33,17 @@ build-std = ["std", "panic_abort"]
 #build-std-features = ["panic_immediate_abort"] # Only necessary if building against ESP-IDF tag `v4.3.2` (the minimum supported version)
 
 [env]
+# Select ESP IDF version in embuild's format described here:
+# https://github.com/esp-rs/embuild/blob/v0.30.4/src/espidf.rs#L585
```

Comment:
The version format is described in details in the README file of `esp-idf-sys`, so you probably need to refer to the README instead (https://github.com/esp-rs/esp-idf-sys).

Reply:
Thank you for pointing me there! Updated with [7521051](https://github.com/ivmarkov/rust-esp32-std-demo/pull/136/commits/752105194ba056f40a9f56b3987f1aa8b85bc422).